float64[] x
float64[] y
float64[] z
float64[] vx
float64[] vy
float64[] t
float64[] angle
float64[] t_ini
bool stop_world
---
bool result